Sources/Valet/MigrationError.swift

```diff
@@ -25,14 +25,14 @@ import Foundation
 public enum MigrationError: Int, CaseIterable, CustomStringConvertible, Error, Equatable {
     /// Migration failed because the keychain query was not valid.
     case invalidQuery
-    /// Migration failed because a key in the query result could not be read.
-    case keyInQueryResultInvalid
-    /// Migration failed because some data in the query result could not be read.
-    case dataInQueryResultInvalid
-    /// Migration failed because two keys with the same value were found in the keychain.
-    case duplicateKeyInQueryResult
-    /// Migration failed because a key in the keychain duplicates a key already managed by Valet.
-    case keyInQueryResultAlreadyExistsInValet
+    /// Migration failed because a key staged for migration was empty.
```

Comment:
I think we should leave this comment a bit more vague, since it can be invalid for reasons other than being empty (e.g. the key is a type other than a String, so we can’t parse it).

Reply:
That's entirely fair. Updating.